todo_app/templates/index.html

```diff
@@ -20,23 +20,45 @@ <h2>Items</h2>
         </tr>
       </thead>
       <tbody>
-        {% for item in items %}
+        {% for item in view_model.items %}
```

Comment:
Minor, but worth noting that since you're not using the filters in the template (e.g. `view_model.done_items`) then they only ever get called in the tests so they're very much an exercise-artifact rather than useful. I'm not worried about that, but we should be conscious that this is one of those scenarios where we're definitely diverging from a real codebase in having unit tests against unused code 😄